common/tests/unittests/test_model_filter.py

```diff
@@ -6,162 +6,32 @@
-            None,
-            {"dummy": {}}
-        ),
-        (
-            {"dummy": {"dummy": 1}},
-            ["dummy"],
-            None,
-            {"dummy": {"dummy": 1}}
-        ),
-        (
-            {"dummy": {}},
-            ["foo"],
-            None,
-            {}
-        ),
-        (
-            {"dummy": {}, "foo": {}},
-            ["foo"],
-            None,
-            {"foo": {}}
-        ),
-        (
-            {},
-            ["foo"],
-            None,
-            {}
-        ),
-        (
-            {"dummy": {}},
-            None,
-            ["dummy"],
-            {}
-        ),
-        (
-            {"dummy": {"dummy": 1}},
-            None,
-            ["dummy"],
-            {}
-        ),
-        (
-            {"dummy": {}},
-            None,
-            ["foo"],
-            {"dummy": {}}
-        ),
-        (
-            {"dummy": {}, "foo": {}},
-            None,
-            ["foo"],
-            {"dummy": {}}
-        ),
-        (
-            {},
-            None,
-            ["foo"],
-            {}
-        ),
-        (
-            {"dummy": {}},
-            ["dummy"],
-            ["dummy"],
-            {"dummy": {}}
-        ),
-        (
-            {"dummy": {"dummy": 1}},
-            ["dummy"],
-            ["dummy"],
-            {"dummy": {"dummy": 1}}
-        ),
-        (
-            {"dummy": {}},
-            ["foo"],
-            ["dummy"],
-            {}
-        ),
-        (
-            {"dummy": {}, "foo": {}},
-            ["foo"],
-            ["dummy"],
-            {"foo": {}}
-        ),
-        (
-            {},
-            ["foo"],
-            ["foo"],
-            {}
-        ),
-        (
-            {"dummy": {}},
-            None,
-            None,
-            {"dummy": {}}
-        ),
-        (
-            ["dummy"],
-            ["dummy"],
-            None,
-            ["dummy"]
-        ),
-        (
-            ["dummy"],
-            ["foo"],
-            None,
-            []
-        ),
-        (
-            ["dummy", "foo"],
-            ["foo"],
-            None,
-            ["foo"]
-        ),
-        (
-            [],
-            ["foo"],
-            None,
-            []
-        ),
-        (
-            ["dummy"],
-            None,
-            ["foo"],
-            ["dummy"]
-        ),
-        (
-            [],
-            None,
-            ["foo"],
-            []
-        ),
-        (
-            ["dummy"],
-            None,
-            ["dummy"],
-            []
-        ),
-        (
-            ["dummy", "foo"],
-            None,
-            ["dummy"],
-            ["foo"]
-        ),
-        (
-            ["dummy"],
-            None,
-            None,
-            ["dummy"]
-        )
+        ({"dummy": {}}, ["dummy"], None, {"dummy": {}}),
+        ({"dummy": {"dummy": 1}}, ["dummy"], None, {"dummy": {"dummy": 1}}),
+        ({"dummy": {}}, ["foo"], None, {}),
```

Comment:
I'm a bit sad about this reformatting, but I suppose it is a small price to pay overall.

Reply:
😂